### ISSUE TITLE
Remove the unused condition from various lessons_chapters_view for Accessibility

### ISSUE THREAD

**Issue by MohitGupta121** [oppia/oppia-android#4909]
Currently we using `viewModel.chapterPlayState != ChapterPlayState.NOT_PLAYABLE_MISSING_PREREQUISITES` condition ForAccessibility which is always True in `lessons_in_progress_chapter_view.xml` , `lessons_no_started_chapter_view.xml`  and always False in `lessons_locked_chapter_view.xml`

So we can remove this condition and directly use the true value.
